backend/src/main/java/com/tourneynizer/tourneynizer/model/UserInfo.java

```diff
@@ -0,0 +1,13 @@
+package com.tourneynizer.tourneynizer.model;
+
+public class UserInfo {
+
+    public int wins, losses, tournaments, matches;
+
+    public UserInfo(int wins, int losses, int tournaments, int matches) {
+        this.wins = wins;
+        this.losses = losses;
+        this.tournaments = tournaments;
+        this.matches = matches;
+    }
+}
```

Comment:
Also how are you going to get all of this data at the same time?

Reply:
Eventually they will be columns in user table, getting them on the fly would be too expensive.

Reply:
But would sql query would give you all this info? Or are you gonna have to make multiple ones to make one single user info object?

Reply:
What sql query will give you all this info or will you have to make multiple queries to make one user info object?